DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml

```diff
@@ -22,209 +22,10 @@
     <h1 class="nhsuk-heading-xl" id="app-page-heading">Centre dashboard</h1>
     <p class="nhsuk-body-m">Welcome <span class="nhsuk-u-font-weight-bold">@Model.FirstName</span>. You are managing <span class="nhsuk-u-font-weight-bold">@Model.CourseCategory</span> courses.</p>
 
-  <ul class="nhsuk-grid-row nhsuk-card-group dashboard-4-1-card-group">
+    <partial name="_Dashboard4x1CardGroup" model="Model"/>

```

Comment:
Nice! I like the partials split but I wonder if we could rename it into something more intuitive - I would probably find it easier to find the partial I want to modify without having to go to the Index page, if we had DashboardTopCardGroup/BottomCardGroup partials? Then, if the style of these section changes to say 3x2 we wouldn't have to rename these either 🤔